src/registry/check.rs

```diff
@@ -51,19 +63,63 @@ pub(crate) fn command(
     // Load the semantic convention registry into a local cache.
     // No parsing errors should be observed.
     let semconv_specs = load_semconv_specs(&registry_path, cache, logger.clone())?;
-
-    if !args.skip_policies {
-        check_policies(
+    let mut policy_engine = if !args.skip_policies {
+        Some(init_policy_engine(
             &registry_path,
             cache,
             &args.policies,
-            &semconv_specs,
-            logger.clone(),
-        )?;
+            args.display_policy_coverage,
+        )?)
+    } else {
+        None
+    };
+
+    if let Some(policy_engine) = policy_engine.as_ref() {
+        // Check the policies against the semantic convention specifications before resolution.
+        // All violations should be captured into an ongoing list of diagnostic messages which
+        // will be combined with the final result of future stages.
+        _ = check_policies(policy_engine, &semconv_specs, logger.clone())
```

Comment:
Nit: There's something about ignoring results that bothers me in this PR.  Can you add to the comment above it WHY ignoring the result is ok here?

Reply:
Fixed in https://github.com/open-telemetry/weaver/pull/214/commits/f28cd820ac47f984ea536111b549d655fe591bb2